### DISCUSSION TITLE
Elementor Support

### DISCUSSION THREAD

**Discussion by stevegiorgi** [wpengine/faustjs#1494] [Q&A]
Would it be possible to include Elementor support?  I believe Elementor requires the_content() to be loaded in a page template so that may pose a problem right there.  I know that Elementor also compiles CSS files dynamically, so we would have to include those in our page routes somehow.

**Comment by theodesp** [comment]
Hey @stevegiorgi we currently do not support Elementor as it's in our plugin conflict list. What exactly are you trying to do that will help serving content in a Headless environment?

I'm moving this ticket into a discussion.

**Reply by theodesp** [reply]
Hey @stevegiorgi have you considered using FSE using an experimental plugin you can check out?

https://github.com/AxeWP/wp-graphql-site-editor

Elementor support is out of our immediate goals at the moment.

**Comment by marinageomedia** [comment]
Hello,
I have a conflict between elementor and elementor pro it causes the post type page not to work, it started I think after I updated the plugins
https://hot.geo-media.co.il/highlights/losing-alice/
This the error that I get from my server support:
_09-Aug-2023 09:14:38 UTC] PHP Fatal error:  Uncaught Error: Object of class WP_Error could not be converted to string in /home/hotgeomediaco/public_html/wp-includes/kses.php:1735
Stack trace:
#0 /home/hotgeomediaco/public_html/wp-includes/kses.php(1735): preg_replace('/[\\x00-\\x08\\x0B...', '', Object(WP_Error))
#1 /home/hotgeomediaco/public_html/wp-includes/kses.php(752): wp_kses_no_null(Object(WP_Error), Array)
#2 /home/hotgeomediaco/public_html/wp-includes/kses.php(2157): wp_kses(Object(WP_Error), 'post')
#3 /home/hotgeomediaco/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/post-terms.php(108): wp_kses_post(Object(WP_Error))
#4 /home/hotgeomediaco/public_html/wp-content/plugins/elementor/core/dynamic-tags/tag.php(35): ElementorPro\Modules\DynamicTags\Tags\Post_Terms->render()
#5 /home/hotgeomediaco/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Tag->get_content()
#6 [internal function]: Elementor\Core\DynamicTags\Manager->get_tag_data_content('f9aed6e', 'post-terms', A in /home/hotgeomediaco/public_html/wp-includes/kses.php on line 1735_

**Reply by mindctrl** [reply]
@marinageomedia your stack trace doesn't have anything related to Faust. Do you have more info that points to it possibly being Faust? Can you give an example setup to reproduce?

**Comment by NearlyShake** [comment]
Hey,

I'm also looking for Elementor support and came across this discussion. Basically, it would be a good idea to use Faust.js as a page builder, for example, to build your own widgets with Elementor ( [https://developers.elementor.com/docs/widgets/ ](https://developers.elementor.com/docs/widgets/) ), while pursuing a headless approach on the frontend for maximum performance optimization.

Will something like this be available in the future?

The problem is that many people don't like the Gutenberg editor because of its user-friendliness, and I could imagine combining the two wouldn't be a bad idea.

Thanks in advance for your reply.

**Comment by colinmurphy** [comment]
@NearlyShake 

Thank you so much for your feedback and contribution 🚀 

Currently Faust does not support Elementor out of the box.  We are currently going to start reviewing Faust which will lead to us creating a new roadmap for Faust. This will be made public and also open to contributions and feedback from the community.

We will definitely take your feedback into consideration when creating the roadmap.

Please let us know if you have any questions at all.

**Comment by colinmurphy** [comment]
@NearlyShake 

Just to follow up we have released an RFC yesterday on the future of Faust. Any feedback or suggestions would be greatly appreciated - https://github.com/wpengine/faustjs/discussions/2140
